packages/backend/src/Metrics.ts

```diff
@@ -58,6 +60,13 @@ export class Metrics {
         'uncertaintyBuffer',
       ],
     })
+
+    this.tvlHistogram = this.createHistogram({
+      name: 'tvl_sync_histogram',
```

Comment:
I think this deserves a better description. 
`tvl_sync_histogram` -> `tvl_sync_duration_histogram`
`Histogram showing tvl core sync speed` -> `Histogram showing TVL sync duration`